src/components/monitor/monitor.jsx

```diff
@@ -13,43 +19,74 @@ const categories = {
     list: '#FC662C'
 };
 
+const types = {
+    default: DefaultMonitor,
+    large: LargeMonitor
+};
+
 const MonitorComponent = props => (
-    <Draggable
-        bounds="parent"
-        defaultClassNameDragging={styles.dragging}
-        onStop={props.onDragEnd}
-    >
-        <Box
-            className={styles.monitor}
-            componentRef={props.componentRef}
+    <ContextMenuTrigger id={`monitor-${props.label}`}>
+        <Draggable
+            bounds=".monitor-overlay" // Class for monitor container
+            defaultClassNameDragging={styles.dragging}
+            onStop={props.onDragEnd}
         >
-            <Box className={styles.label}>
-                {props.label}
-            </Box>
             <Box
-                className={styles.value}
-                style={{background: categories[props.category]}}
+                className={styles.monitorContainer}
+                componentRef={props.componentRef}
+                onDoubleClick={props.onNextType}
             >
-                {props.value}
+                {types[props.type]({
+                    categoryColor: categories[props.category],
+                    label: props.label,
+                    value: props.value
+                })}
             </Box>
-        </Box>
-    </Draggable>
+        </Draggable>
+        <ContextMenu id={`monitor-${props.label}`}>
+            <MenuItem onClick={props.onSetTypeToDefault}>
+                <FormattedMessage
+                    defaultMessage="normal readout"
```

Comment:
I noticed that our context menus have lower-cased words and Blockly's are all capitalized.

Reply:
Yeah I was just directly bringing them over from Scratch2